### DISCUSSION TITLE
Double Switch Single Motor

### DISCUSSION THREAD

**Discussion by Sebo848** [Novakasa/brickrail#177] [General]
I did a quick mock-up in bricklink studio.io of a single motor dual switch using the same approach I am currently deploying with my single TB switches - it is 100% doable.  

haha it looks more like some sort of offshore oil-rig. I am using Trixbrix switches and the ones that are back to back are on the second level of the layout - ideally I would hide this inside some sort of building but that is for later refinement.

The red and orange parts are to simulate the interface parts of the track switch 

 
<img width="291" alt="Screenshot 2024-07-21 at 18 23 13" src="https://github.com/user-attachments/assets/f92943a9-b015-4302-ae96-f2ef9fa3fa86">

I haven't tried the vertically arranged switches yet but I'm sure using a similar approach of technic axles and gears it would it absolutely  possible too.

**Comment by Novakasa** [comment]
Looks good! I am also using trixbrix switches, and I once also experimented with mechanically linking them. Here's a pic:
![image](https://github.com/user-attachments/assets/8cb75e70-f525-4ea2-8dd0-fd026eae8b78)

Btw, I've worked on my solution for motorizing a single trixbrix switch for quite some time and published the most recent version on rebrickable here:
https://rebrickable.com/mocs/MOC-164996/Novakasa/motorization-mechanism-for-trixbrix-switches/#details

From your picture it's not quite apparent how the mechanism works, but would definitely be interested in it.

**Reply by Sebo848** [reply]
I think your solution is more sophisticated  - what I have is probably too crude right now. I'm using the Lego linear motor and have tuned them using the Bricklink parameters so this seems to work quite well 

 
<img width="585" alt="Screenshot 2024-07-22 at 21 13 42" src="https://github.com/user-attachments/assets/5eeff593-0192-4fed-bc14-70f157524dac">

<img width="260" alt="Screenshot 2024-07-22 at 19 00 44" src="https://github.com/user-attachments/assets/d2b8214e-622d-4013-93c2-fbe819264a25">

I've only built the single version right now - this one only exists in studio for now
